lib/vtls/rustls.c

```diff
@@ -505,10 +505,9 @@ cr_get_selected_ciphers(struct Curl_easy *data,
   if(!ciphers12) {
     /* Add default TLSv1.2 ciphers to selection */
     for(j = 0; j < default_len; j++) {
-      struct rustls_str s;
       entry = rustls_default_crypto_provider_ciphersuites_get(j);
-      s = rustls_supported_ciphersuite_get_name(entry);
-      if(s.len >= 5 && strncmp(s.data, "TLS13", 5) == 0)
+      if(rustls_supported_ciphersuite_protocol_version(entry) ==
+          RUSTLS_TLS_VERSION_TLSV1_3)
```

Comment:
there's another one of these further up in this function.

Reply:
(but, the inverse)

Reply:
Oops, good catch. Thank you. I was over-focused on the skip logic that was updated in 14840 and missed this. Fixed :ballot_box_with_check: